experimental/adaptive-tool/server/lg/src/buildinFunctions.ts

```diff
@@ -217,7 +217,7 @@ export const buildInfunctionsMap: Map<string, FunctionEntity> = new Map<string,
   [
     'contains',
     new FunctionEntity(
-      ['collection: stirng|Array|Map', 'value: stirng|Array|Map'],
+      ['collection: string|Array|Map', 'value: stirng|Array|Map'],
```

Comment:
>stirng [](start = 47, length = 6)

string

Reply:
Sorry for the delay.
Fixed here: https://github.com/microsoft/BotBuilder-Samples/pull/3110